modules/iam-identity/src/main/java/com/ibm/cloud/platform_services/iam_identity/v1/IamIdentity.java

```diff
@@ -96,7 +103,7 @@ public class IamIdentity extends BaseService {
   /**
    * Default service endpoint URL.
    */
-  public static final String DEFAULT_SERVICE_URL = "https://iam.cloud.ibm.com";
+  public static final String DEFAULT_SERVICE_URL = "https://iam.test.cloud.ibm.com";
```

Comment:
You must have used a non-production version of the API definition to generate this code.  You will need to use the production version instead (sorry).